### ISSUE TITLE
A11y / Sensei / Learn - Duplicate navigation for lesson nav

### ISSUE THREAD

**Issue by joedolson** [WordPress/Learn#2782]
## Type of feedback
//dev

## Description

There is a screen-reader-hidden region immediately above the visible lesson navigation buttons that are mostly duplicates of the visible buttons. They are inside a `nav` region, so they are available as a navigation landmark, but the navigation landmark has no accessible name. Additionally, the screen reader text hidden area contains a duplicate and *disabled* 'Completed' button.

Both the link and the button are still available to screen reader users, so screen reader users have up to two additional links and a disabled button in addition to the visible equivalents. 

It'll be even more confusing, since the first 'Completed' button will also be marked as disabled, but the second one is not. 

Furthermore, the second button is an interactive control that is not disabled, but does not do anything. This is confusing for all users. It should just be text, and not be an interactive element. Most likely, this information should be in the H1 heading on the page, so that a user gets that information immediately, rather than needing to find it at the end of the lesson.

## Screenshots or screen recording (optional)
Screenshot shows the container as it would appear with the `screen-reader-text` class removed. 

![image](https://github.com/user-attachments/assets/d8143cd0-6396-4e5a-85b7-030d045fe143)

**Comment by joedolson** [comment]
Slight correction - the 'Completed' button does have an action when the lesson has not yet been completed. In that case, the button says 'Complete' and marks the course as completed and moves on. However, it should not still be a *button* once the function of the button is obsolete.

**Comment by ryelle** [comment]
It looks like Sensei adds these screen-reader-only elements assuming the lesson won't have its own links there (I traced it back to https://github.com/Automattic/sensei/issues/4738). Since we do have the "Completed" and "Next lesson", it sounds like we can just get rid of the nav & first "completed" button? Leaving only the visible content.

The visible "Completed" button isn't _actually_ a button, though it does appear interactive for a mouse user, that could be fixed too.
